tests/components/maxcube/test_maxcube_binary_sensor.py

```diff
@@ -4,19 +4,23 @@
 from maxcube.cube import MaxCube
 from maxcube.windowshutter import MaxWindowShutter
 
-from homeassistant.components.binary_sensor import DEVICE_CLASS_WINDOW
+from homeassistant.components.binary_sensor import (
+    BinarySensorDeviceClass,
+)
```

Comment:
```suggestion
from homeassistant.components.binary_sensor import BinarySensorDeviceClass
```

Reply:
strange that `python3 -m black homeassistant/components/maxcube/` didn't fix that.

https://github.com/home-assistant/core/pull/58699/commits/6619ad2846520824fd9b0fc96069e7c55d715dee